src/cloud_integration.py

```diff
@@ -65,7 +76,7 @@ def integrate(self, event: ops.EventBase):
             cloud.enable_autoscaling_readonly()
             cloud.enable_instance_modification()
             cloud.enable_region_readonly()
-        elif cloud_name == "gcp":
+        elif cloud_name == "gce":
```

Comment:
similar comment as to k-w; we're double sure the charm knows this cloud as `gce` and not `gcp`?

Reply:
yep... `get_cloud_name` is handled by [this library](https://github.com/charmed-kubernetes/charm-lib-interface-external-cloud-provider/blob/main/charms/interface_external_cloud_provider.py#L13)